repl/repl.go

```diff
@@ -1170,6 +1213,7 @@ var builtin = [...]commandDesc{
 	{"json", []string{}, "set output format to JSON"},
 	{"pretty", []string{}, "set output format to pretty"},
 	{"trace", []string{}, "toggle full trace"},
+	{"metrics", []string{}, "toggle profiling"},
```

Comment:
so for now, leave metrics (parse, compile, eval) in the REPL, but back off on profiling until we get the details right?